artifact/scripter.go

```diff
@@ -43,7 +43,6 @@ func (s *Scripts) Add(path string) error {
 	if s.names == nil {
 		s.names = make(map[string]string)
 	}
-
```

Comment:
There are some whitespace changes here that don't really belong here.

It's ok to do cleanups if you see something which is worth changing, but then they should be in their own commit.